include/logger.h

```diff
@@ -1,85 +1,40 @@
 #ifndef NEWSBOAT_LOGGER_H_
 #define NEWSBOAT_LOGGER_H_
 
-#include <fstream>
-#include <mutex>
-
 #include "config.h"
 #include "strprintf.h"
 
 namespace newsboat {
+	enum class Level;
+}
 
-/* Be sure to update loglevel_str array in src/logger.cpp if you change this
- * enum. */
-enum class Level { NONE = 0, USERERROR, CRITICAL, ERROR, WARN, INFO, DEBUG };
+#ifdef __cplusplus
```

Comment:
Is there a point in checking if we compile with a C++ compiler, if the file can only be parsed by a C++ compiler? I think we can just leave the `ifdef __cplusplus` check.